_includes/build/requesting_data_all_three.html

```diff
@@ -170,8 +170,8 @@ <h3 class="ds-c-alert__heading"> Files expire after 24 hours.</h3>
 	</div>
 
 	<p>
-		The "Accept-Encoding: gzip" header is optional, but will return a significantly smaller (40X smaller) file with a faster download speed. You	 will need to uncompress the file after downloading if you supply this header. When downloading multiple files, you can further speed up the process by downloading multiple files concurrently.
+		The "Accept-Encoding: gzip" header is optional, but will return a significantly smaller (40X smaller) file with a faster download speed. Follow our recommended best practices for <a href="#speeding-up-downloads" class="in-text__link">speeding up downloads</a>.
```

Comment:
Can we decrease to ~30X? In tests, compression ratios are closer to 30x than 40.

Reply:
e0b9c03